### DISCUSSION TITLE
How to display Rewarded interstitial?

### DISCUSSION THREAD

**Discussion by aliresana** [invertase/react-native-firebase#5243] [Q&A]
Rewarded interstitial is a new format of ad units. It is currently in beta release and users aren’t required to opt-in to view it. Is It possible to use the rnfirebase AdMob to show this kind of ad?

**Comment by mikehardy** [comment]
Likely not, there are major deprecations for admob that we still aren't handling, support for beta formats is lower priority than handling breaking changes so we need a PR that supports those prior to anything about new features: #5150
